tyrant/constants.py

```diff
@@ -120,6 +120,11 @@
     "To power a flashlight bulb, you need 500 wired lemons to conduct electricity.",
     "The heaviest lemon was 11 pounds, 9.7 ounces in 2003.",
     "Lemon was a common unisex name in the 1900’s.",
+    "Lemons are nature’s best insect repellent.",
+    "The name “lemon” may have originated from the Middle East",
+    "The Moroccans preserve lemons in jars before they are used in different dishes.",
+    "Lemons turn from green to yellow because of temperature changes.",
+    " A bowl of fresh lemons makes an easy, effortless air freshener.",
```

Comment:
```suggestion
    "A bowl of fresh lemons makes for an easy, effortless air freshener.",
```
Removed evil space, improved wording.